mods/rv/rules/ai.yaml

```diff
@@ -652,6 +659,15 @@ Player:
 			gaweat: 2
 			namisl: 2
 			yappet: 2
+			## AI Tower Rush
+			nalasr-tr: 8
+			tesla-tr: 8
+			naflak-tr: 8
+			gapill-tr: 8
+			nasam-tr: 2
```

Comment:
Why NASAM 2 but NAFLAK 8?

Reply:
Convenient to my AI test, need to let allied more focus on ground to hold back Soviet.